CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+This repository welcomes contributions and corrections. Before submitting a pull request, ensure that you respect these guidelines for SVGs:
+
+  - Include the SVG namespace declaration `xmlns="http://www.w3.org/2000/svg"`. This is not required if the SVG is included inline on HTML5 web pages.
+  - Include a descriptive `<title>` element for accessibility. For example, `<title>GitHub icon</title>`.
+  - Add `aria-labelledby="title"` to the `<svg>` element to improve screen reader support.
+  - Use a `viewBox` of 24 by 24, following [Google’s Material Design guidelines for system icons](https://material.io/guidelines/style/icons.html#icons-system-icons).
```

Comment:
Almost all existing SVG's use a `viewBox` of 16 by 16. The only ones that don't are HTML5 (14x16) and Nucleo (24x24). Why the change?